src/fetch_package.js

```diff
@@ -82,7 +132,20 @@ export default async function fetchPackage(
             }
 
             if (requestStats) {
-                requestStats.packageFetches++;
+                /**
+                 * A little state machine to track what came from the cache.
+                 */
+                if (result._cached === "new") {
+                    result._cached = "no";
+                } else if (result._cached === "no") {
+                    result._cached = "yes";
+                }
```

Comment:
In the method outline below, this code just goes away. (Writing this as a reminder and also to see how GitHub multiline comments handles comments in the same zone as other comments).